tests/src/python/test_qgsserver_wms_getmap.py

```diff
@@ -41,9 +41,10 @@
 
 
 class TestQgsServerWMSGetMap(QgsServerTestBase):
-
     """QGIS Server WMS Tests for GetMap request"""
 
+    #regenerate_reference = True
```

Comment:
What is the purpose of `regenerate_reference`?

Reply:
@pblottiere https://github.com/qgis/QGIS/blob/master/tests/src/python/test_qgsserver.py#L65

Reply:
Ah yes, I forgot that. Thanks!